llrt_modules/src/modules/perf_hooks.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use crate::ModuleInfo;
+use chrono::Utc;
+use llrt_utils::module::export_default;
+use rquickjs::{
+    atom::PredefinedAtom,
+    module::{Declarations, Exports, ModuleDef},
+    prelude::Func,
+    Ctx, Object, Result,
+};
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+pub static TIME_ORIGIN: AtomicUsize = AtomicUsize::new(0);
+
+fn get_time_origin() -> f64 {
+    let time_origin = TIME_ORIGIN.load(Ordering::Relaxed) as f64;
+
+    time_origin / 1e6
+}
+
+fn now() -> f64 {
+    let now = Utc::now().timestamp_nanos_opt().unwrap_or_default() as f64;
+    let started = TIME_ORIGIN.load(Ordering::Relaxed) as f64;
+
+    (now - started) / 1e6
+}
+
+fn to_json(ctx: Ctx<'_>) -> Result<Object<'_>> {
+    let obj = Object::new(ctx.clone())?;
+
+    obj.set("timeOrigin", get_time_origin())?;
+
+    Ok(obj)
+}
+
+pub fn new_performance(ctx: Ctx<'_>) -> Result<Object<'_>> {
+    if TIME_ORIGIN.load(Ordering::Relaxed) == 0 {
+        let time_origin = Utc::now().timestamp_nanos_opt().unwrap_or_default() as usize;
+        TIME_ORIGIN.store(time_origin, Ordering::Relaxed)
+    }

```

Comment:
Move this out and create an "init" function that sets time origin if not already set. Then call this `init` function from vm.rs

Reply:
Since TIME_ORIGIN has been moved to the perf_hooks module, the peformance module can now also be moved to llrt_modules
Added perf_hooks/init function, called when the VM is initialized